test/integration/MetaDataAtom.spec.ts

```diff
@@ -124,32 +124,34 @@ describe('RLAU-572: MetaData in Atoms', () => {
             })
         })
     })
-  
 
-    it('3. should fail with too much metadata', function(done) {
+
+    it('3. should send a valid atom with 65kb of metadata', async () => {
         // X = 1 byte
         let hugeMetaData = ''
         for (let i = 0; i < 65536; i++) {
             hugeMetaData += 'X'
         }
 
-        buildTestAtom({
-            massive: hugeMetaData,
-        }).then((atom) => {
-            nodeConnection.submitAtom(atom).subscribe({
-                error: (e) => { done() },
-                complete: () => { done('Should have failed') },
+        return new Promise((resolve, reject) => {
+            buildTestAtom({
+                massive: hugeMetaData,
+            }).then((atom) => {
+                nodeConnection.submitAtom(atom).subscribe({
+                    error: (e) => { reject(e.message) },
+                    next: () => { resolve() },
```

Comment:
Definitely don't want to resolve on `next` - it's going to fire even before it hits the node with the `PENDING` status, `complete` would work.

However this should really be rejected by the node, could you get a bug report into the RNET board? https://radixdlt.atlassian.net/secure/RapidBoard.jspa?rapidView=76&projectKey=RNET&view=planning

Reply:
Hmm, looked like the other tests used `next`

Reply:
I tried `complete` first but it didn't work